Dockerfile.envoy-tests

```diff
@@ -0,0 +1,12 @@
+FROM fedora:30
```

Comment:
Hmmmm. So this makes _three_ Linux distributions we're using while wrangling Ambassador... 😂